generator/generator-opc-generator/generators/app/index.js

```diff
@@ -0,0 +1,62 @@
+//@ts-nocheck
+'use strict';
+const Generator = require('yeoman-generator');
+const chalk = require('chalk');
+const yosay = require('yosay');
+const fs = require( 'fs' );
+
+module.exports = class extends Generator {
+  prompting() {
+    // Have Yeoman greet the user.
+    this.log(
+      yosay(`Welcome to the super ${chalk.red('opc-generator')} generator!`)
+    );
+
+    const prompts = [
+      {
+        type: 'input',
+        name: 'elementName',
+        message: 'What would you like to call your element? ',
+        default: 'my-element'
+      },
+      {
+        type: 'input',
+        name: 'authorName',
+        message: 'Please enter the author name: '
+      },
+      {
+        type: 'confirm',
+        name: 'installDependencies',
+        message: 'Do you want to install dependencies?'
```

Comment:
IMO remove this question as dependencies should be installed yv default.

Reply:
Installing by default